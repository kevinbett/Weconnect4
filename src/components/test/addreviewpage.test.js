import AddReviewPage from "../AddReviewsPage"
import { shallow } from "enzyme"

describe('Add Review Page',() => { 
    it ('renders add review body', () => { 
        expect(shallow(<AddReviewPage/>)).toMatchSnapshot()
    });
} )
