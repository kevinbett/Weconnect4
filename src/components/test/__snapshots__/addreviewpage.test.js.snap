// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Review Page renders add review body 1`] = `
<div>
  <Navigation />
  <AddReviewForm />
</div>
`;
